lib/jekyll/commands/serve/live_reload_reactor.rb

```diff
@@ -96,11 +76,9 @@ def reload(pages)
               :liveCSS => true,
             }
 
-            Jekyll.logger.debug("LiveReload:", "Reloading #{p.url}")
-            Jekyll.logger.debug(JSON.dump(msg))
-            @websockets.each do |ws|
-              ws.send(JSON.dump(msg))
-            end
+            Jekyll.logger.debug "LiveReload:", "Reloading #{p.url}"
+            Jekyll.logger.debug JSON.dump(msg)
+            @websockets.each { |ws| ws.send(JSON.dump(msg)) }
```

Comment:
@awood does `JSON.dump(msg)` need to be called separately for each websocket or can I store the value of the first dump to a local variable and then use that variable in the block..?

Reply:
You should be able to store the serialized JSON and re-use it.  Not sure why I didn't do that in the first place.  I believe `@websockets` will generally just have one item in it anyway (each item in the list represents a browser that's connected IIRC) unless someone is really excited about livereload.  But it would be better just to dump it to JSON once per `msg`